README.md

```diff
@@ -1,3 +1,6 @@
+# Tourism Kata
+[Demo Site](http://tourism-kata.nilliofy.com/)
```

Comment:
Joe you said that you had a netlify link, but this is a nilliofy link?
Did you mean https://tourism-kata.netlify.app/

Reply:
Ah shoot sorry, will fix now.

Reply:
Perfect, goo work Joe